lib/generators/solidus_braintree/install/templates/app/assets/javascripts/spree/frontend/solidus_braintree/hosted_form.js

```diff
@@ -7,6 +7,7 @@ SolidusBraintree.HostedForm.prototype.initialize = function() {
   this.client = SolidusBraintree.createClient({
     paymentMethodId: this.paymentMethodId,
     useThreeDSecure: (typeof(window.threeDSecureOptions) !== 'undefined'),
+    useDataCollector: true,
```

Comment:
@SyborgStudios I removed the `IS_FRONTEND` const since we now have separate `hosted_form` JS files for frontend and backend.

Reply:
Awesome. I was not happy with it anyways.